packages/engine/Source/Scene/DynamicEnvironmentMapManager.js

```diff
@@ -841,12 +845,50 @@ DynamicEnvironmentMapManager.prototype.destroy = function () {
   return destroyObject(this);
 };
 
+/**
+ * Returns <code>true</code> if dynamic updates are supported in the current WebGL rendering context.
+ * Dynamic updates requires the EXT_color_buffer_float or EXT_color_buffer_half_float extension.
+ *
+ * @param {Scene} contextOption The object containing the rendering context
+ * @returns {boolean} true if supported
+ */
+DynamicEnvironmentMapManager.isDynamicUpdateSupported = function (
+  contextOption,
+) {
```

Comment:
little bit of a nitpick but I think something needs renamed for clarity somewhere.

The type of this argument is `Scene` but it's called `contextOption` but when it's used in the `update` function above it's passed a `frameState`.

It's obviously working fine but the naming seems wrong?

Reply:
Yes, ideally this would be typed as `{Scene|FrameState}`. However, `FrameState` is technically private, and therefor I can't use that without failing the build. 

I can add a private version for internal use only if we're OK with the redundancy. Otherwise, I'll add a comment to explain.

Reply:
Would it be more accurate to just change the type to "any object that has a context property" like `{{context: Context}}` or `{any & {context: Context}}`? I think there's a way to do that in JSDoc. Not a big deal though. I don't think there's a strong need to duplicate the function

Reply:
My concern is that the docs may be a bit confusing for users. I changed the naming to be more scene focused, and will add a comment for my internal usage to explain.